clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -872,7 +923,12 @@ def get_wf(subject_id,
                            name='vol2vol_mask')
 
     if 'SPMSTANDALONE_HOME' in os.environ:
-        tpmnii = os.path.join(str(os.getenv("SPM_HOME")), 'spm12_mcr/spm12/spm12/tpm/TPM.nii')
+        # Path is different between SPM standalone MAC and Linux. First is MAC location
+        if platform.system() == 'Darwin':
+            tpmnii = os.path.join(str(os.getenv("SPMSTANDALONE_HOME")), 'spm12_mcr/spm12/tpm/TPM.nii')
+        elif platform.system() == 'Linux':
+            # Try Linux location
+            tpmnii = os.path.join(str(os.getenv("SPMSTANDALONE_HOME")), 'spm12_mcr/spm12/spm12/tpm/TPM.nii')
```

Comment:
Why don't you use the `get_tpm()` function in `utils/spm.py` ?

Reply:
This function does not seem to work if SPM standalone  only is installed (at least on a Mac machine). See screenshot below
![Screen Shot 2019-09-24 at 15 01 59](https://user-images.githubusercontent.com/49677712/65513879-6d557800-dedc-11e9-9f6c-99e2d8d964eb.png)
All my environment variables are set properly though...
As I said, there is really some work to do to have spm standalone in order.(Maybe this is caused by the newest version of spm standalone, who knows !)

Reply:
See my response [here](https://github.com/aramis-lab/clinica/pull/31#discussion_r327720728).

I close this discussion.